src/test/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/integration/SinkConnectorIT.java

```diff
@@ -0,0 +1,263 @@
+import com.microsoft.azure.cosmosdb.kafka.connect.IntegrationTest;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+import static org.apache.kafka.common.utils.Utils.sleep;
+import static org.sourcelab.kafka.connect.apiclient.request.dto.NewConnectorDefinition.Builder;
+
+/**
+ * Integration tests for the CosmosDB sink connector.
+ * Tests that messages posted to Kafka propogate to CosmosDB.
+ * 
+ * NOTE: Requires a Cosmos DB instance and the docker-compose orchestration to be running.
+ */
+@Category(IntegrationTest.class)
+public class SinkConnectorIT {
+    private static Logger logger = LoggerFactory.getLogger(SinkConnectorIT.class);
+
+    private final String COSMOSDB_SINK_CONNECTOR_NAME = "test-cosmosdb-sink";
+    
+    private String databaseName;
+    private String topic;
+    private Builder connectConfig;
+    private CosmosClient cosmosClient;
+    private CosmosContainer targetContainer;
+    private KafkaConnectClient connectClient;
+    private KafkaProducer<String, JsonNode> producer;
+
+    /**
+     * Load CosmosDB configuration from the connector config JSON and set up CosmosDB client.
+     * Create an embedded Kafka Connect cluster.
+     */
+    @Before
+    public void before() throws URISyntaxException, IOException {
+        // Load the sink.config.json config file
+        URL configFileUrl = SinkConnectorIT.class.getClassLoader().getResource("sink.config.json");
+        JsonNode config = new ObjectMapper().readTree(configFileUrl).get("config");
+        String topicContainerMap = config.get("connect.cosmosdb.containers.topicmap").textValue();
+        topic = StringUtils.substringBefore(topicContainerMap, "#");
+        String containerName = StringUtils.substringAfter(topicContainerMap, "#");
+
+        // Setup Cosmos Client
+        logger.debug("Setting up the Cosmos DB client");
+        cosmosClient = new CosmosClientBuilder()
+                .endpoint(config.get("connect.cosmosdb.connection.endpoint").textValue())
+                .key(config.get("connect.cosmosdb.master.key").textValue())
+                .buildClient();
+        
+        // Create CosmosDB database if not exists
+        databaseName = config.get("connect.cosmosdb.databasename").textValue();
+        cosmosClient.createDatabaseIfNotExists(databaseName);
+        CosmosDatabase targetDatabase = cosmosClient.getDatabase(databaseName);
+
+        // Create Cosmos Container if not exists
+        CosmosContainerProperties containerProperties =
+                new CosmosContainerProperties(containerName, "/id");
```

Comment:
nit: should this be split across 2 lines? if so, then line 91 should probably be split across 2 lines too.